source/D2Common/src/Units/Units.cpp

```diff
@@ -290,6 +290,19 @@ int __stdcall UNITS_GetTargetX(D2UnitStrc* pUnit)
 		return pUnit->pStaticPath->nTargetX;
 
 	case UNIT_PLAYER:
+	{
+		int targetX = PATH_GetTargetX(pUnit->pDynamicPath);
+		int targetY = PATH_GetTargetY(pUnit->pDynamicPath);
+		static int prevTargetX = targetX;
+		static int prevTargetY = targetY;
+		if (prevTargetX != targetX || prevTargetY != targetY)
+		{
+			prevTargetX = targetX;
+			prevTargetY = targetY;
+		}
+
+		targetX += 0;
+	}
```

Comment:
Was this used for debugging purposes?

Reply:
Damn, I mistakenly committed it. Will remove